cognite/client/data_classes/simulators/models.py

```diff
@@ -404,3 +409,275 @@ def _load_list(
         cls, resource: list[dict[str, Any]], cognite_client: CogniteClient | None = None
     ) -> list[SimulatorModelRevisionExternalDependency]:
         return [cls._load(item, cognite_client) for item in resource]
+
+
+@dataclass
+class SimulatorModelRevisionDataObjectEdge(CogniteObject):
+    id: str
+    name: str | None
+    source_id: str
+    target_id: str
+    connection_type: str
+
+    @classmethod
+    def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> Self:
+        return cls(
+            id=resource["id"],
+            name=resource.get("name"),
+            source_id=resource["sourceId"],
+            target_id=resource["targetId"],
+            connection_type=resource["connectionType"],
+        )
+
+    @classmethod
+    def _load_list(
+        cls, resource: list[dict[str, Any]], cognite_client: CogniteClient | None = None
+    ) -> list[SimulatorModelRevisionDataObjectEdge]:
+        return [cls._load(item, cognite_client) for item in resource]
+
+
+@dataclass
+class SimulatorModelRevisionDataThermodynamic(CogniteObject):
+    property_packages: list[str]
+    components: list[str]
+
+    @classmethod
+    def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> Self:
+        return cls(
+            property_packages=resource["propertyPackages"],
+            components=resource["components"],
+        )
+
+
+@dataclass
+class SimulationValueUnitQuantity(CogniteObject):
```

Comment:
```suggestion
class SimulationValueUnitReference(CogniteObject):
    name: str
    quantity: str | None = None
    external_id: str | None = None

    @classmethod
    def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> Self:
        return cls(
            name=resource["name"],
            quantity=resource.get("quantity"),
            external_id=resource.get("externalId"),
        )
```
it's not just quantity